### ISSUE TITLE
Do not use separate script to spawn browser in FAQ items

### ISSUE THREAD

**Issue by Minoru** [newsboat/newsboat#1205]
As mentioned in https://github.com/newsboat/newsboat/pull/1201#discussion_r496241175, our FAQ [suggests using scripts](https://newsboat.org/releases/2.21/docs/faq.html#_after_i_open_a_link_in_the_browser_newsboat_stays_unresponsive_until_the_browser_is_closed) where the command could fit into the setting. After #1201 is merged, we should replace this advice with `browser /usr/bin/firefox %u &`.

**Comment by A1RO** [comment]
I kept in mind this FAQ item in #1201

---

I rephrased this part

> Newsboat has to behave like that in order to accomodate users of text-based browsers like lynx and w3m. It has to relinquish the terminal and stay out of the way, otherwise people won’t be able to see their browsers.

with

> The default behavior of newsboat when running external commands is to use the same terminal for the new application. It is great for a "classical" workflow i.e., when working from a virtual console without a graphical environment. On the other hand, it might seem like a limitation if you would like to keep using newsboat while the command is running. A few simple tricks can be used to work with both newsboat and an external application at the same time. Those are explained below.

plus I mentioned a way to open text-based browsers in a dedicated terminal:

> For text-based console applications, it is best to first launch a terminal
emulator, which will in turn execute the remainder of the browser command.
Standard output do not need to be silenced with this technique.
Here, `alacritty` is used. Of course, you can replace it with your go-to
terminal emulator, just remember to set the "command" or "execute" option
(`-e` in this example). To open `lynx` in a dedicated terminal:

> :set browser "alacritty -e lynx %u &"

---

This part

> 1. Start your browser beforehand. This way, Newsboat will only wait a fraction of a second (while a new tab is being opened).

is also covered here

> Applications like firefox, if already running, will simply open a new tab when receiving a link, then give control back to newsboat. Because of that, newsboat will be unavailable for a split second, then work can resume.

---

I briefly mentioned

> 2. Wrap your browser in a script and use the script as Newsboat’s browser.

here

> The first step to open links outside of newsboat is to set a default browser in your config file. It can be any command that you would normally run, such as a web browser or an executable script.

**Comment by A1RO** [comment]
At this point, if you are fine with #1201 (I can still modify it before merge), this FAQ item can either be removed, or simplified while adding a link to "Separating Browser from Newsboat" for reference. But the same content should not be present in both pages IMHO.

**Comment by Minoru** [comment]
> But the same content should not be present in both pages IMHO.

I fully agree.

> simplified while adding a link to "Separating Browser from Newsboat" for reference

I'd choose that route. I don't mind you doing the changes, just use a separate PR for that — we'll probably get stuck discussing the wording again, and that'll block #1201 for no good reason. Small steps are better :)
